src/contacts/util/SerializationUtil.java

```diff
@@ -0,0 +1,20 @@
+package contacts.util;
+
+import java.io.*;
+
+public class SerializationUtil {
+    public static void serializeObject(Object object, String fileName) throws IOException {
```

Comment:
this method is not used anywhere

Reply:
It is extra method, I have tried use some time